### PR TITLE
[Enhancement](Column) default value for bigint should not has to be quoted

### PR DESCRIPTION
**Pull request by nanfeng1999**
## Proposed changes

Issue Number: open #20670

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by nanfeng1999** [comment]
run buildall

**Review by yiguolei — COMMENTED** [review]
pls add one regression test

File: fe/fe-core/src/main/cup/sql_parser.cup [hunk 1]
```diff
@@ -3450,6 +3450,10 @@ opt_default_value ::=
     {:
         RESULT = new ColumnDef.DefaultValue(true, value);
     :}
+    | KW_DEFAULT INTEGER_LITERAL:value
```
Comment (stalary) [inline fe/fe-core/src/main/cup/sql_parser.cup:3453]: Maybe we can support decimal, too?

**Review by morningman — COMMENTED** [review]
how about decimal and float

File: fe/fe-core/src/main/cup/sql_parser.cup (hunk 1 above)
Comment (nanfeng1999) [reply fe/fe-core/src/main/cup/sql_parser.cup:3453]: ok，I will try it

**Pull request closed without merging**